src/compute/src/logging/compute.rs

```diff
@@ -234,6 +242,12 @@ pub(super) fn construct<A: Allocate>(
                 Datum::UInt64(bucket.try_into().expect("pow too big")),
             ])
         });
+        let shutdown_duration = shutdown_duration.as_collection().map(move |bucket| {
+            Row::pack_slice(&[
+                Datum::UInt64(u64::cast_from(worker_id)),
+                Datum::UInt64(bucket.try_into().expect("pod too big")),
```

Comment:
```suggestion
                Datum::UInt64(bucket.try_into().expect("bucket too big")),
```

Reply:
Ups! Should be "pow", for consistency with the other expect messages.

Reply:
Renamed this and the instance a couple lines above to `bucket`.